client/package.json

```diff
@@ -58,13 +60,13 @@
     "@babel/plugin-transform-runtime": "^7.21.0",
     "@babel/preset-env": "^7.8.7",
     "@babel/preset-react": "^7.8.3",
+    "@storybook/addon-a11y": "^6.5.16",
```

Comment:
Really small nit is that the alphabetical order has changed which could potentially cause a weird conflict considering the dependency updates happening elsewhere.

Reply:
nice catch ! fixed